packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -204,13 +277,113 @@ export async function processOutboundDocumentRetrievalResps({
   }
 }
 
+async function handleOperationOutcomeIssues({
+  issues,
+  requestId,
+  patientId,
+  cxId,
+}: {
+  issues: Issue[];
+  requestId: string;
+  patientId: string;
+  cxId: string;
+}) {
+  const { log } = out(
+    `CQ handleOperationOutcomeIssues - requestId ${requestId}, patient ${patientId}`
+  );
+  const existingFHIRDocRefs: DocumentReference[] = await getDocumentsFromFHIR({
+    cxId,
+    patientId,
+    documentIds: issues.map(issue => issue?.id ?? ""),
+  });
+
+  const transactionBundle: Bundle = {
+    resourceType: "Bundle",
+    type: "transaction",
+    entry: [],
+  };
+
+  for (const docRef of existingFHIRDocRefs) {
+    if (!docRef.id) continue;
+
+    const updatedFHIRDocRef: DocumentReferenceWithId = {
+      ...docRef,
+      id: docRef.id,
+      status: "entered-in-error",
+    };
+
+    const transactionEntry: BundleEntry = {
+      resource: updatedFHIRDocRef,
+      request: {
+        method: "PUT",
+        url: updatedFHIRDocRef.resourceType + "/" + updatedFHIRDocRef.id,
+      },
+    };
+    transactionBundle.entry?.push(transactionEntry);
+  }
+
+  await upsertDocumentsToFHIRServer(cxId, transactionBundle, log);
+}
+
+async function handleDocReferencesDuplicatedInCw({
```

Comment:
hypernit: this function appears in the flow before the one above, so its definition should also be higher up in the file